app/controllers/admin/dashboards_controller.rb

```diff
@@ -25,7 +25,7 @@ def calculate_completion_rate
     "#{rate}%"
   end
 
-  def get_active_courses
+  def active_courses?
```

Comment:
```suggestion
  def active_courses
```
return kiểu boolean mới cần dấu ? ở cuối